.github/workflows/run_analysis.yml

```diff
@@ -153,3 +153,21 @@ jobs:
       if: false
       run: |
         python 07_test_h1.py
+
+    - name: Run 08_test_h2
+      # Always skip: we would need to download data of all subjects
+      if: false
+      run: |
+        python 08_test_h2.py --overwrite=True
```

Comment:
the command line argument `--overwrite` won't work for these scripts, because you didn't include a parser: https://github.com/NomisCiri/eeg_manypipes_arc/blob/97d296ff1331004ea4266799f9e17e0338a1c2bc/03_run_ica.py#L44-L66

I don't think we need this for the hypotheses scripts though :-)

-